wiki/History_of_osu!/April_Fools/es.md

```diff
@@ -0,0 +1,275 @@
+
+## 2017
+
+Como se anunció en un [tuit de peppy](https://twitter.com/ppy/status/848021525663842304), no hubo broma del día de los Inocentes para osu! en 2017 debido al desarrollo de la versión [lazer](/wiki/Client/Release_stream/Lazer) del cliente de osu!.
+
+## 2018
+
+### osu! cookie rotativa
+
+El 1 de abril de 2018, la [osu! cookie](/wiki/Brand_identity_guidelines) en la pantalla del menú principal giraba lentamente en el sentido de las agujas del reloj a medida que pasaba el tiempo, y la galleta en la pantalla de selección de canciones giraba lentamente en el sentido contrario a las agujas del reloj. Si pasa el cursor sobre estas galletas, se agrandarán como de costumbre, pero también hará que giren más rápido.[^osu-cookie-web-reddit][^osu-cookie-web-reddit-2][^osu-cookie-web-reddit-3][^osu-cookie-web-forums][^osu-cookie-web-forums-2]
+
+## 2019
+
+### Efecto de sonido de chica estornudando
+
+Para el día de los Inocentes de 2019, había aproximadamente una probabilidad de 1 en 20 de escuchar el efecto de sonido de una chica estornudando al abrir un mapa.[^sneeze-reddit][^sneeze-reddit-2][^sneeze-forums]
+
+## 2020
+
+### MillhioreF se une a los artistas destacados
+
+[MillhioreF](https://osu.ppy.sh/users/941094), un moderador de osu! desde hace mucho tiempo, desarrollador y jugador de [Easy mod](/wiki/Gameplay/Game_modifier/Easy), se anunció en [una publicación de noticia](https://osu.ppy.sh/home/news/2020-04-01-new-featured-artist-millhioref) como si se hubiese "unido" a la lista de [Artistas destacados](/wiki/People/Featured_Artists) como "Millhiore Firianno Biscotti" el 1 de abril de 2020 con una selección de cinco canciones para empezar:[^irish-fa]
+
+- Waltz o' the Irish
+- The Waltzing Irishman
+- An Irish Waltz
+- A Waltz From The Geographical Region Known as Ireland but Also as Éire
+- There's Gold Beneath Your Waltzing Rainbow (feat. Mismagius)
+
+["MillhioreF - Waltz o' the Irish (MillhioreF)"](https://osu.ppy.sh/beatmapsets/73348#osu/326585) — un mapa de broma de larga duración dentro de la comunidad, también fue [Amado](/wiki/Beatmap/Category#loved) el 31 de marzo de 2020 como parte de esta broma.
+
+### Efecto de sonido de chica estornudando
+
+El día de los Inocentes de 2020 recicló la misma broma del año anterior, lo que provocó una probabilidad de aproximadamente de 1 en 20 de escuchar el efecto de sonido de una chica con un tono alto estornudando al abrir un mapa.[^sneeze-2-reddit][^sneeze-2-reddit-2]
+
+## 2021
+
+### Efecto de sonido de chica estornudando
+
+El día de los Inocentes de 2021 recicló la misma broma de los dos años anteriores, que era una probabilidad de 1 en 20 de escuchar un efecto de sonido de una chica con un tono alto estornudando al abrir un mapa.[^sneeze-2-forums][^sneeze-2-forums-2]
+
+## Referencias
```

Comment:
reference titles need to be localized as well according to the ASC: https://osu.ppy.sh/wiki/es/Article_styling_criteria/Formatting#notas-al-pie

Reply:
I don't understand this one, I didn't move any of the references. Except for reference #1 which doesn't come out directly (but you can see it in Wayback Machine) and #44, the others work.

Should I change the ones that don't work or what should I do?

Reply:
"localized" means that you need to translate the titles of the references (excluding titles of the pages they refer to) to Spanish -- take a look at the article styling criteria I linked. for example, in `Reddit post by u/5522Luca in r/osugame (2017-04-10) "Reminder the Osu! April Fools 2009? This beatmap was ranked."` you would translate the first part but leave the post's topic, `Reminder the Osu! April Fools 2009? This beatmap was ranked`, unchanged -- and it goes for all the references

Reply:
Ah, I noticed that in the other languages, but I thought it was by preference.

I'll do that in a while when I finish the contests. Thanks for the explanation.